### PR TITLE
Implement `jumplist` command

### PR DESCRIPTION
**Pull request by jdujava**
Shows recent jumps in jumplist.

### CODE REVIEW

File: zathura/commands.c [hunk 1]
```diff
@@ -149,6 +149,69 @@ bool cmd_bookmark_open(girara_session_t* session, girara_list_t* argument_list)
...
+    return false;
+  }
+
+  int num_entries = 5;
+
+  const unsigned int argc = girara_list_size(argument_list);
+  if (argc == 1) {
+    const char* value = girara_list_nth(argument_list, 0);
+    if (value != NULL) {
+      if (atoi(value) == 0) {
```
Comment (sebastinas) [inline zathura/commands.c:167]: The result of this operation can be stored and does not need to be repeated.

Also we mostly use `strto*` to convert strings to integers as it has better error handling.

File: zathura/commands.c [hunk 2]
```diff
@@ -149,6 +149,69 @@ bool cmd_bookmark_open(girara_session_t* session, girara_list_t* argument_list)
...
+  girara_list_iterator_t* iter = girara_list_iterator(zathura->jumplist.list);
+  while (girara_list_iterator_has_next(iter)) {
+    girara_list_iterator_next(iter);
+  }
+
+  GString* string = g_string_new(NULL);
+  for (int i = 0; i < num_entries; ++i) {
+    zathura_jump_t* j = girara_list_iterator_data(iter);
+    g_string_append_printf(string, "[%d]: page=<b>%2d</b>, x=%f, y=%f %s\n", zathura->jumplist.size - i - 1, j->page,
+                           j->x, j->y, girara_list_iterator_data(iter) == current_jump ? "(current)" : "");
```
Comment (sebastinas) [inline zathura/commands.c:196]: page and current should be translatable strings.

File: zathura/commands.c [hunk 3]
```diff
@@ -149,6 +149,69 @@ bool cmd_bookmark_open(girara_session_t* session, girara_list_t* argument_list)
...
+
+  if (num_entries <= 0) {
+    return true;
+  }
+
+  zathura_jump_t* current_jump = zathura_jumplist_current(zathura);
+  girara_list_iterator_t* iter = girara_list_iterator(zathura->jumplist.list);
+  while (girara_list_iterator_has_next(iter)) {
+    girara_list_iterator_next(iter);
+  }
```
Comment (sebastinas) [inline zathura/commands.c:190]: This loop and the next loop can be simplified by iterating from `size - 1` down.  The use of iterators should mostly go away.

**Review by sebastinas — CHANGES_REQUESTED** [review]

**Comment by jdujava** [comment]
Tried to implement all of the suggestions. Glad to hear any additional feedback (mainly just trying to adapt already existing code).

File: zathura/commands.c (hunk 2 above)
Comment (jdujava) [reply zathura/commands.c]: Not sure if I regenerated all necessary files correctly. If not, pointers to documentation or necesarry commands would be helpful.

**Pull request merged**